protocols/tests/test_util/test_generate_mock_objects.py

```diff
@@ -56,6 +56,15 @@ def test_clinical_report_cancer(self):
         self.assertTrue(isinstance(test_cir, self.model.ClinicalReportCancer))
         self.assertTrue(test_cir.validate(test_cir.toJsonDict()))
 
+    def test_rd_exit_questionnaire(self):
+        """
+        Ensure generate_mock_objects.get_valid_rd_exit_questionnaire_4_0_0 returns a valid
+        reports_4_0_0.RareDiseaseExitQuestionnaire object
+        """
+        test_cir = generate_mock_objects.get_valid_rd_exit_questionnaire_4_0_0()
+        self.assertTrue(isinstance(test_cir, self.model.RareDiseaseExitQuestionnaire))
```

Comment:
do you want a method like the following??? This won't save so many LOCs, but it may make test code more robust.
```
def verify_mocked_object(object, class):
    self.assertTrue(isinstance(object, class))
    self.assertTrue(object.validate(object.toJsonDict()))
```